tests/edalize_common.py

```diff
@@ -1,21 +1,108 @@
-import os.path
-import tempfile
 from collections import OrderedDict
+import filecmp
+import os.path
+import shutil
+
+import pytest
 
 from edalize import get_edatool
 
+
 tests_dir = os.path.dirname(__file__)
 
+
+class TestFixture:
+    '''A fixture that makes an edalize backend with work_root directory
+
+    Create this object using the make_edalize_test factory fixture. This passes
+    its keyword arguments through to the TestFixture initializer, which
+    supports the following:
+
+    test_name: The name to call the backend. Default: test_<tool_name>_0
```

Comment:
Please add an

```
Args:
  test_name: ...
```

to render the arguments in the docs. You can try it out by switching over to the `doc` directory, and typing `make html`.

Reply:
Turns out that's not enough to get pretty parameter lists. I've updated the patch with the [correct Sphinx syntax](https://sphinx-rtd-tutorial.readthedocs.io/en/latest/docstrings.html), which now looks pretty horrible in the source code. Is that better or worse?

Reply:
It should be enough (and I tested the diff I've posted initially), since edalize uses the napoleon sphinx extension (https://www.sphinx-doc.org/en/master/usage/extensions/napoleon.html). Maybe it needs slightly more indentation?

Reply:
Ahah! Yes, it looks like it was just an indentation problem. Much better :-)